examples/ct/ct_handlers_test.go

```diff
@@ -434,16 +431,13 @@ func makeAddChainRequest(t *testing.T, reqHandlers CTRequestHandlers, body io.Re
 	handler := wrappedAddChainHandler(reqHandlers)
 
 	req, err := http.NewRequest("POST", "http://example.com/ct/v1/add-chain", body)
-	if err != nil {
-		t.Fatalf("test request setup failed: %v", err)
-	}
+
```

Comment:
Nit: There is an abundance of vertical whitespace, but that doesn't bother me much.